netdata-installer.sh

```diff
@@ -259,6 +259,7 @@ while [ -n "${1}" ]; do
     "--enable-ebpf") NETDATA_ENABLE_EBPF=1 ;;
     "--disable-cloud")
       NETDATA_DISABLE_LIBMOSQUITTO=1
+      NETDATA_SKIP_LIBWEBSOCKETS=1
```

Comment:
Nit: Stick to `NEDTAT_DISABLE_XXX`? for consistency?

Reply:
It's not disabling anything though, just skipping trying to bundle libwebsockets.